multinet/multinet.py

```diff
@@ -86,26 +87,47 @@ def create_graph(workspace, graph):
     if missing:
         return (missing, "400 Missing Required Parameters")
 
-    # Validate that all referenced tables exist
-    edges = list(db.db(workspace).collection(edge_table).find({}))
-    invalid_from = set(
-        [
-            edge["_from"].split("/")[0]
-            for edge in edges
-            if edge["_from"].split("/")[0] not in node_tables
-        ]
-    )
-    invalid_to = set(
-        [
-            edge["_to"].split("/")[0]
-            for edge in edges
-            if edge["_to"].split("/")[0] not in node_tables
-        ]
-    )
-
-    if invalid_from or invalid_to:
-        error = {"error": "undefined_tables", "detail": list(invalid_from | invalid_to)}
-        return (error, "400 Edge Table Validation Failed")
+    errors = []
+
+    loadedWorkspace = db.db(workspace)
+    existing_tables = set([x["name"] for x in loadedWorkspace.collections()])
+    edges = loadedWorkspace.collection(edge_table).all()
+
+    # Iterate through each edge and check for undefined tables
+    valid_tables = dict()
+    invalid_tables = set()
+    for edge in edges:
+        nodes = (edge["_from"].split("/"), edge["_to"].split("/"))
```

Comment:
Do we need to validate that these are in the right format? If one of these reads something like `"/123"`, what would happen then?

Reply:
Answer: we don't need to here, because `validate_csv()` in [the CSV uploader](https://github.com/multinet-app/multinet/blob/use_db_iterator/multinet/uploaders/csv.py#L32) does it when the table is created.